src/sdf_tools/utils.py

```diff
@@ -0,0 +1,48 @@
+import numpy as np
+import sdf_tools
+
+
+def compute_2d_sdf_and_gradient(grid_world, sdf_resolution, sdf_origin, frame='/world'):
```

Comment:
'frame' is intended as a TF frame name; I think that current TF objects if you use a leading `/` in frame names.

Reply:
I removed the leading `/` not sure what your comment was trying to say